src/main/scala/cromwell/engine/workflow/WorkflowOptions.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.engine.workflow
+
+import com.typesafe.config.ConfigFactory
+import cromwell.util.{EncryptedBytes, SecretKey, Aes256Cbc}
+import scala.collection.JavaConverters._
+import spray.json._
+import scala.util.{Try, Success, Failure}
+
+object WorkflowOptions {
```

Comment:
Or since this is temporary perhaps just add the iv column to the main DB table. That feels icky but in theory we could just drop the column in the future.

Tagging @mcovarr as he & I were just chatting about this in person but this thread is hard to see in github